site/en/_partials/privacy-sandbox/demos/shared-storage.md

```diff
@@ -0,0 +1,22 @@
+{% Details %}
+{% DetailsSummary %}
+**Shared Storage**
+{% endDetailsSummary %}
+
+{% Columns %}
+{% Column %}
+The Shared Storage API provides storage that can be used cross-site, meaning you store a value at one domain, then read the value from another domain. You can set data freely, but are restricted in how you get data out of storage. This demo illustrates how it works.
+
+[Demo](https://shared-storage-demo.web.app/)
+
+[Docs](/docs/privacy-sandbox/attribution-reporting/shared-storage/)
+
+{% endColumn %}
+{% Column %}
+
+<a href="https://shared-storage-demo.web.app/">{% Img src="image/RtQlPaM9wdhEJGVKR8boMPkWf443/dOFtwOaI8ldFfwgmG5El.png", alt="Shared Storage API 
+ demo.", width="751", height="521" %}</a>
```

Comment:
Is this space (between API and demo) a typo?

Reply:
oh yes!